include/mbedtls/ssl.h

```diff
@@ -2023,6 +2023,40 @@ int mbedtls_ssl_set_cid( mbedtls_ssl_context *ssl,
                          unsigned char const *own_cid,
                          size_t own_cid_len );
 
+/**
+ * \brief              Get information about our request for usage of the CID
```

Comment:
Ok, now I'm confused about what we're trying to do exactly. Should this function return what we requested, or what we negociated? I'll ask in #5184.

Reply:
Ok, seems returning what we requested is indeed what the original reporter asked for. (Which IMO makes this more an optimisation request than a regression fix, but well, we started it, so let's finish it.)